Source/ARTAuth.m

```diff
@@ -275,13 +275,13 @@ - (void)executeTokenRequest:(ARTTokenRequest *)tokenRequest callback:(void (^)(A
 
 - (void)authorise:(ARTTokenParams *)tokenParams options:(ARTAuthOptions *)authOptions callback:(void (^)(ARTTokenDetails *, NSError *))callback {
     BOOL requestNewToken = NO;
-    ARTAuthOptions *mergedOptions = [self mergeOptions:authOptions];
-    [self storeOptions:mergedOptions];
-    ARTTokenParams *currentTokenParams = [self mergeParams:tokenParams];
+    ARTAuthOptions *currentAuthOptions = authOptions ? authOptions : self.options;
+    [self storeOptions:currentAuthOptions];
+    ARTTokenParams *currentTokenParams = tokenParams ? tokenParams : _tokenParams;
     [self storeParams:currentTokenParams];
 
     // Reuse or not reuse the current token
-    if (mergedOptions.force == NO && self.tokenDetails) {
+    if (currentAuthOptions.force == NO && self.tokenDetails) {
```

Comment:
I think you should just remove `force` from this PR and perhaps bundle the change for RSA10d into this as this is only confusing the review of a PR that is in flux between old & new functionality